contracts/index/IndexToken.sol

```diff
@@ -19,24 +19,22 @@ import {
     AggregatorV3Interface,
     IOracleAdapter
 } from "contracts/oracle/Imports.sol";
-import {MetaPoolToken} from "contracts/mapt/MetaPoolToken.sol";
 
-import {IERC4626, IFeePool, ILockingPool, IReservePool} from "./Imports.sol";
+import {IERC4626, IFeeVault, ILockingVault, IReserveVault} from "./Imports.sol";
```

Comment:
As we work on each component, we should go back to using direct imports. As we reduce the dependencies, the value from them is less compared to the more complex contract verification on etherscan and the confusion from having to sometimes do direct imports anyway to avoid a circular dependency.